rpmlint/descriptions/PythonCheck.toml

```diff
@@ -0,0 +1,20 @@
+python-tests-in-package="""
+test/ or tests/ directory in package directory. Tests in %{python_sitelib}/<pkgname> should be packaged in a separate test or tests subpackage."""
+python-doc-in-package="""
+doc/ or docs/ directory in package directory. Documentation should go into %{docdir}, not %{python_sitelib}/<pkgname>"""
```

Comment:
Personally, I think site-packages is the wrong place -- it isn't easily discoverable, and furthermore, it leads to uninstallability bugs when two packages can not be co-installed because they happen to make a doc or docs module under site-packages.

Reply:
Yes, ownign side-packages/doc(s) is bad, but not because it's not in %{docdir} but because it clashes with others.